content/mooreslaw-tutorial.ipynb

```diff
@@ -0,0 +1,807 @@
+{
```

Comment:
<p>I feel like these two sentences</p><p> </p><p><code>You directly compare the number of predicted transistors to the actual number of transistors per microprocessor over the course of manufacturing history. Plot the linear regression results and all of the transistor counts.</code></p><p> </p><p>may be unnecessary from the surrounding text. I think you mention in the paragraph before that you will compare the model with the actual reports, and in the next paragraph you will do the plotting. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
You're right, its redundant. thanks for catching that.